cypress/integration/unit_tests/add_disaster_test.js

```diff
@@ -184,6 +184,35 @@ describe('Unit tests for add_disaster page', () => {
         })
         .then((success) => expect(success).to.be.true);
   });
+
+  it('deletes a disaster', () => {
+    cy.on('window:confirm', () => true);
+
+    const id = '2002-winter';
+    const states = ['DN, WF'];
+
+    cy.document()
```

Comment:
why do you need this cy.document()?

Reply:
Ah sorry, leftover from figuring out how to get the window to work. Removed.